### DISCUSSION TITLE
luckysheet初始化时，单元格配置中带函数的（例如："f": "=SUM(A3:F3)"），为啥还要还要开发者自己去算默认值？这个算不算bug？

### DISCUSSION THREAD

**Discussion by zhangming9502** [dream-num/Luckysheet#535] [Q&A]
![image](https://user-images.githubusercontent.com/67034004/108977763-db9afd00-76c3-11eb-8492-5cee234296ac.png)

**Comment by Hi-Stranger** [comment]
公式链写上 就可以不用自己算

**Comment by hzmings** [comment]
calcChain: [], + luckysheet.refreshFormula(); 了解一下。
